src/Role/MappedParameter/RoleTreeListingParameter.php

```diff
@@ -0,0 +1,38 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\StudioBackendBundle\Role\MappedParameter;
+
+use Symfony\Component\Validator\Constraints\NotBlank;
+use Symfony\Component\Validator\Constraints\PositiveOrZero;
+
+/**
+ * @internal
+ */
+final readonly class RoleTreeListingParameter
+{
+    public function __construct(
+        #[PositiveOrZero]
+        #[NotBlank]
+        private int $parentId = 1
```

Comment:
Do we need a whole class for Role here that just has the parentId in it? 
Not sure if we should make it more generic here.

WDYT?

Reply:
I would say this is a good Idea, since we have this also in the User Section. Than we should make it something like ParentIdParameter. I will take a look.